content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx

```diff
@@ -7,7 +7,13 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add custom methods to your models. We introduced this feature in version 4.7.0.
+<Admonition type="info">
+
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```

Comment:
missing the preview feature flag it was introduced under so people can figure out how to use it with older versions

Reply:
Done!